make/dev/nodeclarity/Dockerfile

```diff
@@ -8,7 +8,14 @@ COPY make/dev/nodeclarity/entrypoint.sh /
 
 WORKDIR /harbor_resources
 
+RUN useradd -m -d /home/node -s /bin/bash node \
```

Comment:
This is not needed as this docker image is for building the ui code only, not for running.